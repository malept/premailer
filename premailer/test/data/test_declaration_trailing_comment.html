<html>
<head>
<title>Title</title>
<style type="text/css">
h1 { color:red; /* trailing comment */ }
</style>
</head>
<body>
<h1>Hi!</h1>
</body>
</html>
