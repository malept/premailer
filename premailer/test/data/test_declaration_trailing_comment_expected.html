<html>
<head>
<title>Title</title>
</head>
<body>
<h1 style="color:red">Hi!</h1>
</body>
</html>
